examples/hybrid/driver.jl

```diff
@@ -119,10 +120,41 @@ end
 
 if config.parsed_args["check_conservation"]
     FT = Spaces.undertype(axes(sol.u[end].c.ρ))
-    @test sum(sol.u[1].c.ρ) ≈ sum(sol.u[end].c.ρ) rtol = 50 * eps(FT)
-    @test sum(sol.u[1].c.ρe_tot) +
-          (p.net_energy_flux_sfc[][] - p.net_energy_flux_toa[][]) ≈
-          sum(sol.u[end].c.ρe_tot) rtol = 100 * eps(FT)
+
+    # energy
+    energy_total = sum(sol.u[end].c.ρe_tot) - p.net_energy_flux_toa[][]
+    energy_atmos_change = sum(sol.u[end].c.ρe_tot) - sum(sol.u[1].c.ρe_tot)
+    sfc = p.atmos.surface_model
+    if sfc isa CA.PrognosticSurfaceTemperature
+        sfc_cρh = sfc.ρ_ocean * sfc.cp_ocean * sfc.depth_ocean
+        energy_total +=
+            horizontal_integral_at_boundary(sol.u[end].sfc.T .* sfc_cρh)
+        energy_surface_change =
+            horizontal_integral_at_boundary(
+                sol.u[end].sfc.T .- sol.u[1].sfc.T,
+            ) * sfc_cρh
+    else
+        energy_surface_change = 0
+    end
+    energy_radiation_input = -p.net_energy_flux_toa[][]
```

Comment:
I think if surface temperature is not prognostic, `energy_radiation_input` needs to take into account `p.net_energy_flux_sfc[][]` as well. Did the atmosphere only energy conservation check in ci pass?

Reply:
Ahh yes, you're right (I was testing it elsewhere before). Now this passes for prescribed surface too.